widget/RazorpayElementsSubscriptionButton.php

```diff
@@ -168,7 +168,22 @@ protected function render() {
 			?>
 			
 			<form>
+			<?php
+			global $wp_version;
+			if($wp_version >= 5.7){
+				wp_print_script_tag(
+						array(
+							"src"      => "https://cdn.razorpay.com/static/widget/subscription-button.js",
+							"data-plugin "       => ($dataPlugin),
```

Comment:
@neera11 can you please add some comment here regarding the if-else condition. Also please make the alignment proper here.